src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -98,8 +98,15 @@ private void OnShutdownCompleted()
         }
 
         #endregion
-        
+
+        #region internal members
+
+        private PulseMaker pulseMaker;
+
```

Comment:
See above.  `PulseMaker` will be a cyclic dependency and should probably live in `WorkspaceModel`.